<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SensorThings API - Connection to server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="lblConnection">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connection details</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frmDetails">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frmDetails {
    border: 1px solid lightgray;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblName">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblURL">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtURL"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
